build.ps1

```diff
@@ -1,13 +1,19 @@
 ﻿param (
   [string]$buildNumber = "0",
   [string]$extensionVersion = "2.0.$buildNumber",
-  [bool]$includeSuffix = $true
+  [bool]$includeSuffix = $true,
+  [string]$devopsPRTitle = ""
```

Comment:
I think that  `build.ps1` shouldn't know anything about the environment (whether appveyor or devops). Instead, let's pass in `$bypassPackaging` as a parameter. Then the different CI pieces that call this need to figure it out.

The reason is that this script is still meant to be used locally as well to build your packages if you need to test them. The more CI-specific stuff we add internally will make that more confusing.